theme/templates/accounts/profile.html

```diff
@@ -4,78 +4,516 @@
-                {% else %}
-                    <li><strong>Email:</strong> (<a href="/accounts/login/">Log in to send email</a>)</li>
-                {% endif %}
-            {% endif %}
-            {% if profile.cv %}
-            <li><a href="{{ profile.cv.url }}">Download a curriculum vitae</a></li>
-            {% endif %}
-        {% else %}
-            <li>User has chosen to make their profile information private.</li>
-        {% endif %}
-    </ul>
-</div>
-
-<div class="col-md-5 text-justify">
-<h2>About {{ u.first_name }}</h2>{{ profile.details }}</div>
-
-<div class="col-md-4">
-<h2>Resources</h2>
-    <ul class="nav nav-stacked">
-    {% for res in resources %}
-        <li><a href="{{ res.get_absolute_url }}">{{ res.title }}</a></li>
-    {% empty %}
-        <li>This user has no Resources, yet.</li>
-    {% endfor %}
-    </ul>
-</div>
-</div>
-
-
-{% endwith %}
+                                            {% if res.metadata.description %}
+                                                <div class="activity-description col-sm-12"
+                                                        {% if profile.picture and profile.picture.url %}style="margin-left:15px;" {% endif %}>
+                                                    <p class="description">
+                                                        <strong>ABSTRACT: </strong>{{ res.metadata.description|linebreaks }}
+                                                    </p>
+                                                </div>
+                                                <div class="show-more-btn btn-default" title="Show abstract">···</div>
+                                            {% endif %}
+                                        </div>
+                                        {% endif %}
+                                    {% endfor %}
+
+                                    {% else %}
+                                        <span class="text-muted">This user has no recent activity.</span>
+                                    {% endif %}
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="col-md-5">
+                            {#        DESCRIPTION       #}
+                            {% if profile.details %}
+                            <div class="panel panel-default">
+                                <div class="panel-heading">
+                                    <h3 class="panel-title">About {{ u.first_name }}</h3>
+                                </div>
+                                <div class="panel-body" style="color:#757575;">
+                                    {{ profile.details }}
+                                </div>
+                            </div>
+                            {% endif %}
+
+                            {#  CONTACT #}
```

Comment:
This has been addressed, but somehow is not reflected in the comment here. Can be verified by viewing full changes.